pkgs/development/python-modules/txzmq/default.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pyzmq
+, twisted }:
+
+buildPythonPackage rec {
+  pname = "txzmq";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit version;
+    pname = "txZMQ";
+    sha256 = "sha256-jWB9C/CcqUYAuOQvByHb5D7lOgRwGCNErHrOfljcYXc=";
+  };
+
+  propagatedBuildInputs = [
+    pyzmq
+    twisted
+  ];
+
```

Looks that that `pytestCheckHook` could work for the tests. 

```suggestion
  checkInputs = [
    pytestCheckHook
  ];
```

---

Done. It still skips 9 tests though.